Documentation/unix-instructions.md

```diff
@@ -1,5 +1,5 @@
-Building CoreFX on Linux
```

Comment:
Seems I actually missed a few references. Will correct that and push a new commit.